src/components/List/ItemGroup/Trigger.tsx

```diff
@@ -0,0 +1,25 @@
+import { AccordionTrigger } from "@radix-ui/react-accordion";
+import { ReactNode } from "react";
+
+import { Sprinkles } from "~/theme";
+import { ChervonDownIcon } from "~/components";
+
+import { Item } from "../Item";
+
+import { icon } from "./styles.css";
+
+type ItemGroupTriggerProps = Pick<
+  Sprinkles,
+  "gap" | "paddingY" | "paddingX" | "borderRadius" | "justifyContent"
+> & {
+  children: ReactNode;
+};
+
+export const Trigger = ({ children, ...rest }: ItemGroupTriggerProps) => (
```

Comment:
question: shouldn;t trigger has default values for ` paddingX={4}
        paddingY={4}
        borderRadius={3}
        justifyContent="space-between"` ?

Reply:
Defaults will be applied when those components are used e.g in Dashboard.